gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala

```diff
@@ -236,6 +236,7 @@ object ExpressionMappings {
     Sig[CheckOverflow](CHECK_OVERFLOW),
     Sig[MakeDecimal](MAKE_DECIMAL),
     Sig[PromotePrecision](PROMOTE_PRECISION),
+    Sig[NaNvl](NANVL),
```

Comment:
Do we need this mapping if we replace nanvl with if and isnan?

Reply:
This is needed because of check in https://github.com/oap-project/gluten/blob/main/gluten-core/src/main/scala/io/glutenproject/expression/ExpressionConverter.scala#L120.
Thanks.

Reply:
Add this func to mappings for velox only now. Thanks.